.golangci.yml

```diff
@@ -143,6 +143,7 @@ linters-settings:
       - G107
       - G404
       - G601
+      - G115  
```

Comment:
we don't want to turn the rule completely

Reply:
@yurishkuro Then, should i use 
```
//nolint: gosec // G115
```
Instead?

Reply:
yes